lib/perl/Genome/Qc/Tool.pm

```diff
@@ -0,0 +1,45 @@
+package Genome::Qc::Tool;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Qc::Tool {
```

Comment:
If this class is abstract (as the method definitions seem to indicate), you could add `is_abstract =>1` to the class definition to enforce that.